localisation/00_zuwc_united_localisation.csv

```diff
@@ -2978,6 +2978,17 @@ GetRegenerationReason_Loa;Due to Loa's blessings;;Wegen der Segen der Loa;;;;;;;
 regenerate_body_part_cooldown;Recently Used Regeneration;;Hat k�rzlich Regeneration genutzt.;;;;;;;;;;;x
 EVTOPTA_WCTRO_500_eunuch;I feel tingly down there.;;Es kribbelt da unten.;;;;;;;;;;;x
 EVTDESC_WCTRO_500;You immerse your body in a pool with shadowflame decoction. The water is quite warm, but you feel like you are burning. Holding back a cry, you continue to sit absolutely still. Finally you feel a slight tingling sensation. You get out of the pool and feel renewed like a newborn baby.;;Ihr taucht Euren K�rper in ein Becken mit Abkochung der Schattenflamme. Das Wasser ist ziemlich warm, aber Ihr f�hlt Euch, als w�rdet Ihr brennen. Ihr haltet einen Schrei zur�ck und sitzt weiterhin absolut still da. Schlie�lich versp�rt Ihr ein leichtes Kribbeln. Ihr steigt aus dem Becken und f�hlt Euch wie ein neugeborenes Baby.;;;;;;;;;;;x
+WAR_NAME_TOTAL_INVASION_OF;$ORDER$$FIRST_ADJ$ Great Invasion of $OTHER_TITLE$;;$ORDER$$FIRST_ADJ$ Gro�e Invasion von $OTHER_TITLE$;;;;;;;;;;;x
+EVTDESC_WCIMM_20;You notice that your courtiers speak badly of you more often lately. No wonder, for you have ruled for an unusually long time. Both you and your people have grown weary of your rule.;;Ihr stellt fest, dass Eure H�flinge in letzter Zeit vermehrt schlecht �ber Euch sprechen. Kein Wunder, denn Ihr regiert schon au�ergew�hnlich lange. Sowohl Ihr als auch Euer Volk sind Eurer Herrschaft �berdr�ssig geworden.;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCIMM_20;You have noticed that your courtiers are speaking badly of you more often lately. It's no wonder, for you have ruled for an unusually long time. Both you and your people have grown weary of your rule.;;Ihr stellt fest, dass Eure H�flinge in letzter Zeit vermehrt schlecht �ber Euch sprechen. Kein Wunder, denn Ihr regiert schon au�ergew�hnlich lange. Sowohl Ihr als auch Euer Volk sind Eurer Herrschaft �berdr�ssig geworden.;;;;;;;;;;;x
```

Reply:
Done in 1bfabf901ae20d7b320846f02dcec1de51a0c4a6.